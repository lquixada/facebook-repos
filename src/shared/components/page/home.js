import React from 'react'
import styled from 'styled-components'

import {Content, Section, SectionContainer} from '../grid'
import {flex} from '../../helpers'

export const HomePage = () => (
  <Section>
    <SectionContainer>
      <Content>
        <P>
          Type a github user in the input above and hit Enter.
        </P>
      </Content>
    </SectionContainer>
  </Section>
)

const P = styled.p`
  ${flex.display()}
  ${flex.middle()} 
  ${flex.center()}
  height: 50rem;
  border: .7rem dashed #eee;
  border-radius: .6rem;
  font-size: 1.4rem;
  letter-spacing: -.1px;
`
