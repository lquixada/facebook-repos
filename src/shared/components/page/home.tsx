import React from "react";
import {Helmet} from "react-helmet";
import styled from "styled-components";

import {flex} from "../../helpers";
import {Content, Section, SectionContainer} from "../grid";

export const HomePage = () => (
  <Section>
    <Helmet>
      <title>Github Repos</title>
    </Helmet>

    <SectionContainer>
      <Content>
        <P>
          Type a github user in the input above and hit Enter.
        </P>
      </Content>
    </SectionContainer>
  </Section>
);

const P = styled.p`
  ${flex.display()}
  ${flex.middle()}
  ${flex.center()}
  height: 50rem;
  border: .7rem dashed #eee;
  border-radius: .6rem;
  font-size: 1.4rem;
  letter-spacing: -.1px;
`;
