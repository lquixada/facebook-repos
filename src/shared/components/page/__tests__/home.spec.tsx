import {shallow} from 'enzyme'
import React from 'react'

import {HomePage} from '../home'

describe('<HomePage />', () => {
  it('matches the snapshot', () => {
    const component = shallow(<HomePage />)
    expect(component).toMatchSnapshot()
  })
})
