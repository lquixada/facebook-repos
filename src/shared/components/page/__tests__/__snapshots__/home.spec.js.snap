// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> matches the snapshot 1`] = `
<styled.section>
  <styled.div>
    <styled.article>
      <styled.p>
        Type a github user in the input above and hit Enter.
      </styled.p>
    </styled.article>
  </styled.div>
</styled.section>
`;
